langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/AzureOpenAiModelName.java

```diff
@@ -4,30 +4,31 @@ public class AzureOpenAiModelName {
 
     // Use with AzureOpenAiChatModel and AzureOpenAiStreamingChatModel
     public static final String GPT_3_5_TURBO = "gpt-3.5-turbo"; // alias for the latest model
-    public static final String GPT_3_5_TURBO_0301 = "gpt-3.5-turbo-0301"; // 4k context
     public static final String GPT_3_5_TURBO_0613 = "gpt-3.5-turbo-0613"; // 4k context, functions
+    public static final String GPT_3_5_TURBO_0125 = "gpt-3.5-turbo-0125"; // 4k context, functions
```

Comment:
You might want to split this class into multiple enums: one per model type, like OpenAiChatModelName, OpenAiEmbeddingModelName, etc. These enums can be then accepted by model builder for convenience (in addition to strings).